modules/programs/vdirsyncer.nix

```diff
@@ -92,7 +92,7 @@ let
     else if (n == "passwordPrompt") then
       ''password.fetch = ["prompt", "${v}"]''
     else if (n == "verify") then
-      "verify = ${if v then "true" else "false"}"
+      ''verify = "${v}"''
```

Comment:
out of scope but this function is weird and unmaintainable :/ we should have a more transparent mapping !